docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  oauth_client:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "${PORT}:${PORT}"
+    environment:
+      - CLIENT_ID=${CLIENT_ID}
+      - WELL_KNOWN_URL=${WELL_KNOWN_URL}
+      - PORT=${PORT}
+      - HOST=${HOST}
+      - REDIRECT_HOST=${REDIRECT_HOST}
+      - CERT_PATH=${CERT_PATH}
+      - KEY_PATH=${KEY_PATH}
+      - ROOT_CA=${ROOT_CA}
+      - USE_PYRON=${USE_PYRON}
+      - X_SSL_CERT_HASH=${X_SSL_CERT_HASH}
+      - RESOURCE_URL=${RESOURCE_URL}
+      - INJECT_CERT_MODE=${INJECT_CERT_MODE}
```

Comment:
there is no need to explicitly list all environment variables, docker-compose should read `.env` file by default

Reply:
It does read the .env file and that is why the `${}` assignments are there. Without those it won't get them from what I can tell. Perhaps I am doing something wrong or misunderstanding what you mean and you could show me what you mean?

Reply:
Try https://github.com/cloudentity/openbanking-quickstart/blob/master/docker-compose.yaml#L36